src/components/content/heroes/aurora/weather-easter-egg/README.md

```diff
@@ -0,0 +1,113 @@
+# Weather Easter Egg
```

Comment:
Thank you for taking the idea of writing a README.md and taking it into even higher levels than expected. It's now a great example how to document a specific feature ✨